src/redux/reservations/reservationsSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const URL = 'https://dummyjson.com/products/?limit=5';
+
+// create the thunk
+export const fetchUserReservations = createAsyncThunk(
+  'reservations/fetchUserReservations',
+  async () => {
+    const response = await fetch(URL).then(
+      (data) => data.json(),
+    );
+    return response;
+  },
+);
```

Comment:
- [ ] Please be consistent about the method you choose to work with `async` functions. You are mixing together `async-await` and `catch-then` for the same `fetch` operation.

Reply:
Is this a problem?

Reply:
Sorry, I should have flagged it as optional since is not a bug.

This is a suggestion for readability, it would be better to stick to one way after all `async-await` is just sugar syntax to avoid using `then-catch`.